### ISSUE TITLE
DeepSSM in Studio - Data Augementation

### ISSUE THREAD

**Issue by jadie1** [SCIInstitute/ShapeWorks#1371]
Need a tab in studio for DeepSSM data augmentation 

Parameters:

- Number of samples
- Number of PCA dimensions
- Percent variability preserved
- Sampler type: Gaussian, Mixture of Gaussian, or KDE

Run Augmentation button - progress bar at bottom

Display (after running):

- Parallel violin plot
- Examples of real and augmented data

**Comment by akenmorris** [comment]
Are we planning on supporting multiple domains with DeepSSM?  If so, how will that work?

**Comment by sheryjoe** [comment]
@akenmorris Yes, it is still under research though. For now, let's assume single domain.
